src/core/algorithms/gfd/gfd.cpp

```diff
@@ -9,3 +14,78 @@ std::string Gfd::ToString() {
     parser::graph_parser::WriteGfd(gfd_stream, *this);
     return gfd_stream.str();
 }

```

Comment:
WriteGfd function calls write_graphviz function which only handles non-constant graph. I tried to figure out how to bypass this but my attempts were not successful. I would be grateful if you could tell me how to do it

Reply:
`graph_parser.cpp`:

```
160  void WriteGfd(std::ostream& stream, Gfd& result) {
161      WriteLiterals(stream, result.GetPremises());
162      WriteLiterals(stream, result.GetConclusion());
163      graph_t pattern = result.GetPattern();
164      WriteGraph(stream, pattern);
165  };
```

Your aim is to change `result` type --> `Gfd const&`, you can do this since you mark all used methods as const